data/brands/tourism/hotel.json

```diff
@@ -920,25 +920,29 @@
     {
       "displayName": "Gaylord",
       "id": "gaylord-779ccb",
-      "locationSet": {"include": ["001"]},
+      "locationSet": {"include": ["us"]},
       "tags": {
         "brand": "Gaylord",
         "brand:wikidata": "Q3099664",
         "brand:wikipedia": "en:Gaylord Hotels",
         "name": "Gaylord",
+        "official_name": "Gaylord Hotels",
```

Comment:
Since we preserve names for hotels, I don't really think it makes sense to have an `official_name` tag for htoels
```suggestion
```